csrc/host_ir/jit.cpp

```diff
@@ -760,14 +862,28 @@ class HostIrCompileDispatcher : public OptInDispatch {
 
     // Convert input TensorViews to void pointers and get tensor pointers
     llvm::SmallVector<llvm::Value*, 1> input_tensors;
```

Comment:
```suggestion
    llvm::SmallVector<llvm::Value*, 1> inputs;
```
Kernel inputs can be scalars.